wgs_somatic-run-wrapper.py

```diff
@@ -165,15 +166,19 @@ def wrapper():
 
    # next step here is to actually start the pipeline with these arguments
 
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--instrument', help='novaseq_687_gc or novaseq_A01736', required=True)
+    args = parser.parse_args()
 
+    wrapper(args.instrument)
 
 if __name__ == '__main__':
     try:
-        wrapper()
+        main()
     except KeyboardInterrupt:
         pass
-    except Exception:
-        format_exc = traceback.format_exc()
-        logger.error(format_exc)
+    #except Exception:
+        #format_exc = traceback.format_exc()
+        #logger.error(format_exc)
```

Comment:
Why commented?

Reply:
because i haven't handled it yet. will handle later